packages/material-ui/src/Icon/Icon.js

```diff
@@ -66,6 +66,8 @@ const Icon = React.forwardRef(function Icon(props, ref) {
     <Component
       className={clsx(
         'material-icons',
+        // Prevent translations of the content which breaks the CSS logic.
+        'notranslate',
```

Comment:
Do we still need this today? The SO answer you shared that recommended `class="notranslate"` contains outdated links and has last been edited 4 years ago. The `class` might no longer be necessary.

Reply:
Yes, it's still necessary, the icon are broken without.

Reply:
So why add the translate attribute then? As far as I can tell Firefox and IE11 don't support it anyway.

Reply:
The transtate attribute was edge for the future. This seems to be in the specification, so hopefully if there are other cases like this one, we will be covered by the attribute. No strong preferences, we could remove it.

Reply:
We generally don't ship legacy and old features if they do the same thing. Either we use the modern features and advise to polyfill or we inline the polyfill. Both is just confusing.

Reply:
OK sounds fair, +1 for removing the attribute